app/src/main/java/com/github/se/polyfit/MainActivity.kt

```diff
@@ -57,9 +57,10 @@ class MainActivity : ComponentActivity() {
           composable(Route.Register) { LoginScreen(navigation::navigateToHome) }
           composable(Route.AddMeal + "/{mId}") { backStackEntry ->
             val mealId = backStackEntry.arguments?.getString("mId")
+            // If we allow edits from other screens, we will have to modify how we choose goForward
```

Comment:
maybe we can provide another parameter? something like /home if you want to navigate back home and /recap if you want that? But we could leave that until its needed